lib/msal-node/README.md

```diff
@@ -1,50 +1,154 @@
+- [Protected web API: App registration](https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-protected-web-api-app-registration)
+
+## Installation
+
+### Via NPM:
+```javascript
+npm install @azure/msal-node
 ```
-TBD: Add lerna bootstrap to build common/node in one step
 
-## Local Development
+##  Usage
 
-Below is a list of commands you will probably find useful.
+### MSAL basics
+- Initialize a Public Client Application(./docs/initialize-public-client-application.md)
+- [Configuration](./docs/configuration.md)
+- [Request](./docs/request.md)
+- [Response](lib/msal-common/docs/Response.md)
 
-### `npm start`
+## Samples
+There are multiple [samples](samples/msal-node-samples) included in the repository that use MSAL Node to acquire tokens. These samples are currently used for manual testing, and are not meant to be a reference of best practices, therefore use judgement and do not blindly copy this code to any production applications.
 
-Runs the project in development/watch mode. Your project will be rebuilt upon changes. TSDX has a special logger for you convenience. Error messages are pretty printed and formatted for compatibility VS Code's Problems tab.
+- [msal-node-auth-code](samples/msal-node-samples/msal-node-auth-code): Express app using OAuth2.0 authorization code flow.
+- [msal-node-device-code](samples/msal-node-samples/msal-node-device-code): Command line app using OAuth 2.0 device code flow.
+- [msal-node-refresh-token](samples/msal-node-samples/msal-node-refresh-token): Command line app using OAuth 2.0 refresh flow.
+- [msal-node-silent-flow](samples/msal-node-samples/msal-node-silent-flow): Express app using OAuth2.0 authorization code flow and refresh token flow to demonstrate silent retrieval of tokens when already logged in or when the app provides a in-disk cache for Single sign on experience
+- msal-node-extensions (Coming soon): Uses the msal-extensions library to write the MSAL in-memory token cache to a disk.
 
-<img src="https://user-images.githubusercontent.com/4060187/52168303-574d3a00-26f6-11e9-9f3b-71dbec9ebfcb.gif" width="600" />
+## Build and Test
 
-Your library will be rebuilt if you make edits.
+- If you don't have [lerna](https://github.com/lerna/lerna) installed, run `npm install -g lerna`
+- Run `lerna bootstrap` from anywhere within `microsoft-authentication-library-for-js.git`.
+- Navigate to `microsoft-authentication-library-for-js/lib/msal-common` and run `npm run build`
+- Navigate to `microsoft-authentication-library-for-js/lib/msal-node` and run `npm run build`
 
-### `npm run build`
+```javascript
+// Change to the msal-node package directory
+cd lib/msal-common/
+
+// To run build only for node package
+npm run build
+
+// Change to the msal-node package directory
+cd lib/msal-node/
+
+// To run build only for node package
+npm run build
 
+// to link msal-node and msal-common packages
+lerna bootstrap
```

Comment:
lerna bootstrap should be the first thing.

Reply:
Done